virtcontainers/device/api/interface.go

```diff
@@ -49,13 +49,20 @@ type Device interface {
 	// DeviceType indicates which kind of device it is
 	// e.g. block, vfio or vhost user
 	DeviceType() config.DeviceType
+	// HostPath return host path of device
+	GetHostPath() string
 	// GetDeviceInfo returns device specific data used for hotplugging by hypervisor
 	// Caller could cast the return value to device specific struct
 	// e.g. Block device returns *config.BlockDrive and
 	// vfio device returns []*config.VFIODev
 	GetDeviceInfo() interface{}
 	// IsAttached checks if the device is attached
-	IsAttached() bool
+	GetAttachCount() int
+
+	// Reference adds one reference to device
+	Reference() int
```

Comment:
Please can you document here what the return values are for these two new functions?

Reply:
Ah, good point!